src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-features-count/ProteinFeaturesCount.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+
+import {
+  ProteinStats,
+  fetchProteinSummaryStats
+} from 'src/content/app/entity-viewer/shared/rest/rest-data-fetchers/proteinData';
+
+import structuresIcon from 'EBI-Icon-fonts/EBI-Conceptual/static/svg/structures.svg';
+import ligandsIcon from 'EBI-Icon-fonts/EBI-Conceptual/static/svg/chemical.svg';
+import interactionsIcon from 'EBI-Icon-fonts/EBI-Conceptual/static/svg/systems.svg';
+import annotationsIcon from 'EBI-Icon-fonts/EBI-Conceptual/static/svg/cross-domain.svg';
```

Comment:
This is interesting, although I am a little worried that, according to the readme, the ebi-icon-fonts package was not intended to be used like this. Any change in the folder structure ebi-icon-fonts project, and our code will break. Is there any benefit to using this package rather than copying the four icons to our repository?

Reply:
I would also prefer to copy them over to our project, however I didn't do that yet due to possible licence concerns. We need to give attribution to the files that we are using in their project since it is under [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)

Reply:
Add this to either LICENSE or NOTICE file (or both)? For example, the LICENSE file in ensembl-webcode contains information about licenses for special packages. @andrewyatz will likely have an opinion about how to do this.